apps/frontend/src/renderer/components/NewProjectScreen.tsx

```
@@ -0,0 +1,777 @@
+                  <div className="flex items-center gap-2 mb-3">
+                    <Clock className="h-5 w-5 text-muted-foreground" />
+                    <h3 className="font-medium text-foreground">Recent Projects</h3>
+                  </div>
+                  {recentProjects.length > 0 ? (
+                    <div className="space-y-1 max-h-[180px] overflow-y-auto">
+                        {recentProjects.map((project) => (
+                          <button
+                            key={project.id}
+                            onClick={() => onSelectRecentProject(project.id)}
+                            className="w-full flex items-center gap-2 rounded-md px-2 py-2 text-left transition-colors hover:bg-accent/50 group"
+                          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)